pkg/manager/cron/cron.go

```diff
@@ -41,8 +41,23 @@ func (c Cron) processEdits() {
 				voteThreshold = 1
 			}
 
-			var err error
-			if e.VoteCount >= voteThreshold {
+			editVotes, err := c.rfp.Repo().Edit().GetVotes(e.ID)
+			if err != nil {
+				logger.Errorf("Edit cronjob failed to fetch edit votes for edit %s: %s", e.ID.String(), err.Error())
+				return err
```

Comment:
I'm not entirely sure that if there's an error, it will continue processing the other edits, or if it will all just crash.